extend.php

```diff
@@ -26,7 +26,8 @@
 
     (new Extend\Settings())
         ->default('signature.maximum_char_limit', 500)
-        ->default('signature.maximum_image_count', 2),
+        ->default('signature.maximum_image_count', 2)
+        ->serializeToForum('allowInlineEditing', 'signature.allow_inline_editing', 'boolval', false),
```

Comment:
```suggestion
        ->default('signature.allow_inline_editing', false)
        ->serializeToForum('allowInlineEditing', 'signature.allow_inline_editing', 'boolval'),
```

Minor thing, but setting the default like this is safer, as this way a backend calll to get the setting value is guaranteed to have a default, rather than `null`.